restli-server/src/main/java/com/linkedin/restli/internal/server/response/ResponseUtils.java

```diff
@@ -252,16 +252,17 @@ public static RestException buildRestException(RestLiResponseException restLiRes
     return new RestException(restResponse, cause==null ? null : cause.toString(), cause, writableStackTrace);
   }
 
-  public static StreamException buildStreamException(RestLiResponseException restLiResponseException, StreamDataCodec codec)
+  public static StreamException buildStreamException(RestLiResponseException restLiResponseException, ContentType contentType)
```

Comment:
This is a potentially backward incompatible change since it is a public method. I suggest creating an overloaded method and deprecating this method.

Reply:
aah.. thanks , let me update the RB